c/optionals/system.c

```diff
@@ -1,5 +1,31 @@
 #include "system.h"
 
+#define SYS_OK 0
+#define SYS_NOTOK -1
+
+static Value setCWDNative(int argCount, Value *args) {
+    if (argCount != 1) {
+        runtimeError("setcwd() takes 1 argument (%d given)", argCount);
+        return EMPTY_VAL;
+    }
+
+    if (!IS_STRING(args[0])) {
+        runtimeError("setcwd() argument must be a string");
+        return EMPTY_VAL;
+    }
+
+    char *dir = AS_STRING(args[0])->chars;
```

Comment:
For future reference, there is a macro that "exchanges" a string object (ObjString struct) into a char pointer (char *) `AS_CSTRING` found here: https://github.com/Jason2605/Dictu/blob/develop/c/object.h#L37